pkg/cmd/psa/cmd.go

```diff
@@ -88,7 +88,7 @@ func NewCmdPSA(parentName string, streams genericclioptions.IOStreams) *cobra.Co
 	fs := cmd.Flags()
 	o.configFlags.AddFlags(fs)
 	o.printFlags.AddFlags(&cmd)
-	fs.StringVar(&o.level, "level", "restricted", "The PodSecurity level to check against.")
+	fs.StringVar(&o.level, "level", "", "The PodSecurity level to check against.")
```

Comment:
Why ist the default now empty string and what ramifications does that default value have? (place a blurb maybe in the cli help)

Reply:
sgtm